test/app-tap/module_api.c

```diff
@@ -284,6 +284,29 @@ test_fiber(lua_State *L)
 	return 1;
 }
 
+int fiber_set_arg_test_func(va_list va)
```

Comment:
Please, put function return type on a separate line and make the function `static` - it is never accessed outside of this .c file.

Reply:
done